svgdigitizer/electrochemistry/cv.py

```diff
@@ -122,10 +123,24 @@ class CV:
 
         >>> cv.plot()
 
-    The properties of the original plot can be returned as a dict::
-
-        >>> cv.metadata
-        {'figure description': {'type': 'digitized', 'measurement type': 'CV', 'scan rate': {'value': 50.0, 'unit': 'V / s'}, 'potential scale': {'unit': 'mV', 'reference': 'RHE'}, 'current': {'unit': 'uA / cm2'}, 'comment': ''}}
+    The properties of the original plot and the dataframe can be returned as a dict::
+
+        >>> cv.metadata  # doctest: +NORMALIZE_WHITESPACE
+        {'figure description': {'version': 1,
+          'type': 'digitized',
+          'simultaneous measurements': '',
```

Comment:
Should have an empty list of strings here?
```suggestion
          'simultaneous measurements': [],
```